website/search/elastic_search.py

```diff
@@ -292,11 +292,6 @@ def load_parent(parent_id):
         parent_info['url'] = parent.url
         parent_info['is_registration'] = parent.is_registration
         parent_info['id'] = parent._id
-    else:
```

Comment:
Since the `else` clause is being removed, this function could be simplified a bit.
```
def load_parent(parent_id):
    parent = AbstractNode.load(parent_id)
    if parent and parent.is_public:
        return {
            'title': ...,
            'url': ...,
            'id': ...,
            'is_registation': ...,            
        }
    return None
```

Reply:
Solved in 82a37b2. Thanks for the tips.